lib/Security/UserRegisterPolicyProvider.php

```diff
@@ -1,22 +0,0 @@
-<?php
-/**
- * This file is part of the eZ RepositoryForms package.
- *
- * @copyright Copyright (C) eZ Systems AS. All rights reserved.
- * @license For full copyright and license information view LICENSE file distributed with this source code.
- */
-namespace EzSystems\RepositoryForms\Security;
-
-use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\ConfigBuilderInterface;
-use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Security\PolicyProvider\PolicyProviderInterface;
-
-/**
- * Adds the user/register policy.
- */
-class UserRegisterPolicyProvider implements PolicyProviderInterface
```

Comment:
As @webhdx said: Let's mark `UserRegisterPolicyProvider` as deprecated and do not register it. Similar changes has been introduced in https://github.com/ezsystems/ezpublish-kernel/pull/2273